crates/bevy_render/src/lib.rs

```diff
@@ -519,3 +535,27 @@ fn apply_extract_commands(render_world: &mut World) {
             .apply_deferred(render_world);
     });
 }
+
+/// All [`ViewTargetFormat`]s supported by the current [`RenderDevice`]
+#[derive(Resource, Clone, Debug)]
+pub struct SupportedViewTargetFormats {
```

Comment:
```suggestion
pub struct SupportedViewTargetFormats {
pub struct ViewTargetFormats {
```

Can we just call this `ViewTargetFormats`. A little less wordy and something being a valid view target format implies it's supported.